tests/plugin.js

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
Isn't this test file a bit redundant? :thinking: I mean, we already testing if a component has been initialized and destroyed with `tests/integration.js` and `tests/component.js`. WDYT?

Reply:
This file tests only `plugin.install` method. When this method gets broken all or most of the other tests fails. However I'd keep this one because it's assertion might be the best clue of what wrong.